### ISSUE TITLE
Renal tubular epithelial necrosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5001]
Definition:Coagulative necrosis of tubular epithelial cells

**Comment by nicolevasilevsky** [comment]
We could add this logical def:

'has part' some 
    ('having extra processual parts'
     and ('inheres in part of' some 'kidney cortex tubule cell')
     and (towards some 'necrotic cell death')
     and ('has modifier' some abnormal))


I think this is the correct CL term:
CL_1000615 'kidney cortex tubule cell'

Note - this is a child of 'renal cortical epithelial cell'

**Comment by pnrobinson** [comment]
@avizrosenberg
@yongqunh  
@beckysteck

Shouldn't this term be the same as the existing HPO term:
Acute tubular necrosis
https://hpo.jax.org/app/browse/term/HP:0008682

**Comment by avizrosenberg** [comment]
Synonymous, but our definition is a histomorphological definition

**Comment by pnrobinson** [comment]
I think that the best thing is to retain the existing term but to update the definition and make the current label a synonym.

**Comment by pnrobinson** [comment]
@avizrosenberg
I have added the following definition etc to the term. Please reopen this issue if they require revision.

primary label:Renal tubular epithelial necrosis
Synonym:Acute tubular necrosis
Definition: Coagulative necrosis of tubular epithelial cells, defined as cells with increased cytoplasmic eosinophilia and nucleus that has a condensed chromatin pattern with fuzzy nuclear contour or has barely visible nuclear basophilic staining. The extent of cortical tubular necrosis is scored semiquantitatively as none, mild (less than 25% tubules with necrosis), moderate (25-50 percent), and severe (over 50%).
Comment: Renal tubular epithelial necrosis is a histomorphological definition that corresponds to the clinical term of acture tubular necrosis. Acute death of renal tubular cells that usually results from prolonged renal ischemia, nephrotoxins, or sepsis. The clinical course of acute tubular necrosis may be divided into initiation, maintenance, and recovery phases.
